executor/ddl.go

```diff
@@ -152,6 +154,44 @@ func (e *DDLExec) executeCreateTable(s *ast.CreateTableStmt) error {
 	return errors.Trace(err)
 }
 
+func (e *DDLExec) executeCreateView(s *ast.CreateViewStmt) error {
+	ident := ast.Ident{Schema: s.ViewName.Schema, Name: s.ViewName.Name}
+	// SelectFields represents the field expression text in SelectStmt and
+	// saves it as a string to establish the mapping relationship of ColList
+	var selectFields []string
+	selectstmt := s.Select.(*ast.SelectStmt)
+	fields := selectstmt.Fields.Fields
+	for _, field := range fields {
+		if field.WildCard == nil {
```

Comment:
`WildCard` means `select *` , you should unfold it. It seems your test is too few.
Why not compile it to a plan and use plan's schema directly ?

Reply:
Because I need compile the select statement when View Query, if it contains wild card, the plan's schema know the column info, so I need to store the WildCard or plan's schema when create view.